assets/sass/components/_shaded-list.scss

```diff
@@ -0,0 +1,10 @@
+.refer-and-monitor__shaded-list-item {
```

Comment:
You can use `app-inset-text--grey` here - I added it in my branch where I added the new action plan styles (see https://github.com/ministryofjustice/hmpps-interventions-ui/blob/3b48c13868b0dfa56d38f389a771ef9c64c4b6cc/server/routes/serviceProviderReferrals/reviewActionPlanView.ts#L17)